lib/util.js

```diff
@@ -145,6 +147,43 @@ util.secret = function() {
   return string;
 };
 
+util.spawnWithSpinner = async function (command, options = {
+  spinnerMessage: 'Processing...',
+  codeMessage: 'Spawn process error code:'
+}) {
+  if (!command) {
+    throw new Error('No command provided to spawnWithSpinner');
+  }
+
+  const spinner = ora({
+    text: options.spinnerMessage,
+    interval: 100,
+    isEnabled: true
+  });
+
+  return new Promise((resolve, reject) => {
+    try {
+      const installer = spawn(command, {
+        shell: true
```

Comment:
Sorry, I should have asked this before: will the user ever see the output? That's especially important if there's an error.

Reply:
Yes, it's logged here https://github.com/apostrophecms/cli/pull/14/files/4d6bbfb8f4bf4b5d76712784a2da911d255ff76a..52586ebc5934eba202a9e09c74975ed09699027f#diff-98638a4ebf5a3e8155511395d90744a4e069d1a55960da5a9d4c25dc51d07602L75